addons/persistence/infinispan-persistence-addon/src/main/java/org/kie/kogito/infinispan/ProtoStreamObjectMarshallingStrategy.java

```diff
@@ -66,6 +62,7 @@ public boolean accept(Object object) {
         return serializationContext.canMarshall(object.getClass());
     }
 
+
```

Comment:
Try to be mindful on these extra empty line added for no benefit.

Reply:
It is outdated and not a part of changed files.